### PR TITLE
Update [2.-hosted-services.md] aqua deploy_service syntax

### PR DESCRIPTION
**Pull request by ghost**
Was following the tutorial and the syntax to deploy a wasm module is outdated, this is what worked for me.

### CODE REVIEW

File: quick-start/2.-hosted-services.md [hunk 1]
```diff
@@ -172,13 +172,21 @@ Which gets us a list of network peers:
...
+aqua key create
+```
+
+Replace `<YOUR_SECRET_KEY>` with the generated secret key and while in the `quickstart/2-hosted-services` directory run:
+
+```bash
+aqua remote deploy_service \
+    --addr /dns4/kras-04.fluence.dev/tcp/19001/wss/p2p/12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi \
+    --sk <YOUR_SECRET_KEY> \
+    --data-path configs/hello_world_deployment_cfg.json \
```
Comment (monoid) [inline quick-start/2.-hosted-services.md:188]: I believe it should be `--config-path`.

Comment (Azephiar) [reply quick-start/2.-hosted-services.md:188]: If I try to deploy a service using `--config-path` I get `Unexpected option: --config-path`

Comment (monoid) [reply quick-start/2.-hosted-services.md:188]: It seems we have different versions of the `aqua`:
```
$ aqua --version
0.7.2-307
```

Compare with version list at https://www.npmjs.com/package/@fluencelabs/aqua

Comment (boneyard93501) [reply quick-start/2.-hosted-services.md:188]: it is `--config-path` and yes, `--sk` is not optional anymore. @Azephiar please update and we can merge.

**Review by boneyard93501 — APPROVED** [review]

**Pull request merged**